src/modules/common/normalize.js

```diff
@@ -0,0 +1,7 @@
+// @flow
```

Comment:
I think a more descriptive name would be `normalize-search-string`. Else it is not clear in which way something is normalized.